### DISCUSSION TITLE
Uninstall Errors

### DISCUSSION THREAD

**Discussion by mdavidovichHst** [intersystems/git-source-control#185] [Q&A] [answered]
Tried to uninstall using 'zpm uninstall git-source-control' and got the following error:

```
git-source-control]    Clean FAILURE
ERROR! ObjectScript error: <CLASS DOES NOT EXIST>%GetClass+5^%ZPM.PackageManager.Developer.Extension.SourceControl.Interface.1 *SourceControl.Git.Extension
NAMESPACE>zpm "install git-source-control" 
```
When I open studio I get the message:

![image](https://user-images.githubusercontent.com/103590728/163257441-be7e9b66-5360-4163-8615-16cd5c5ec9f8.png)

Tried to install again and got errors:

```
NAMESPACE>zpm "install git-source-control"
 
ERROR! ObjectScript error: <CLASS DOES NOT EXIST>%GetClass+5^%ZPM.PackageManager.Developer.Extension.SourceControl.Interface.1 *SourceControl.Git.Extension
  > ERROR #6315: Errors reporting importing XML subelement in file 'C:\InterSystems\IRISHealth\Mgr\.modules\CLARITI\git-source
control\1.2.0\module.xml' at line '3' offset '43', skipping this item.
```

Does anyone have any ideas of what going on and how I can fix this?  I would like to know how to get a clean uninstall but also be able to install it again if needed.

**Comment by isc-tleavitt** [comment, marked as answer]
@mdavidovichHst try:
do ##class(%Studio.SourceControl.Interface).SourceControlClassSet("")

**Reply by mdavidovichHst** [reply]
Thanks, Tim.  Worked perfect.  What did that end up doing?

**Reply by isc-tleavitt** [reply]
See https://github.com/intersystems/git-source-control/issues/187 - this is just a bug. :)
